README.md

```diff
@@ -1 +1,31 @@
-# ReallSkill JavaScript Tasks
+#Lodash training
+
+##Summary
+
+Training tasks for Lodash.
+
+##Goal
```

Comment:
At the beginning - I don't know what I have to do. Implement lodash function? Use the lodash function? I need explanation (best will be with example).

Maybe we should also add information about methods used in this task?